.gulp/tasks/tests.js

```diff
@@ -16,9 +16,6 @@ module.exports = function (gulp, config) {
                 // Run jasmine under istanbul
                 gulp.src(config.test.src)
                     .pipe(jasmine({ verbose: true, errorOnFail: true, includeStackTrace: false }))
-                    .on('error', (err) => {
```

Comment:
Is this now a built in functionality?

Reply:
Look at line above line, i'm hoping it was fixed at some point as it was a regression causing it to ignore errors at one point. If we have tests fail and it doesn't error the PR then we find out.  Gulp is next on chopping block anyway.